go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/layer5io/smps
```

Comment:
Does this line have to be `module github.com/layer5io/service-mesh-performance-specification` or can we abbreviate with `smps`? The abbreviation is nice.

Reply:
I stuck with the abbreviation because it's shorter.. it's easier to use that ways..

Reply:
Sounds great. I wasn't sure whether it was material (whether the reference needed to be exact).